quartz_solar_forecast/inverters/enphase.py

```diff
@@ -1,24 +1,48 @@
 import http.client
 import os
+from typing import Optional
+
 import pandas as pd
 import json
 import base64
 from datetime import datetime, timedelta, timezone
 
-from dotenv import load_dotenv
+from urllib.parse import urlencode
+
+from quartz_solar_forecast.inverters.inverter import AbstractInverter
+from pydantic import Field
+from pydantic_settings import BaseSettings
```

Comment:
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
quartz-solar-forecast 1.0.65 requires pydantic==2.6.2, but you have pydantic 2.8.2 which is incompatible.
Successfully installed pydantic-2.8.2 pydantic-core-2.20.1 pydantic_settings-2.4.0

Can you update requirements.txt with the appropriate versions?